frontend/elements/src/entry/home/sections/products-section.ts

```diff
@@ -0,0 +1,58 @@
+import { MEDIA_UI } from '@utils/path';
+import "@elements/entry/home/TOSORT/column-list";
+import "@elements/entry/home/social-networks";
+import { LitElement, html, css, customElement, property } from 'lit-element';
+@customElement('products-section')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+    h3{
+      font-family: Poppins;
+      font-size: 20px;
+      font-weight: 800;   
+      color:#ffffff;
+    
+    }
+   main{
+     
+   }
+   
+   ul{
+    list-style-type: none;
+    margin:0;
+    padding:0;
+   }
+
+ 
+ 
+
+  .list{
+    display:block;
+    margin-top:8px;
+ }
+  
+
+    `];
+  }
+
+
+ 
+ 
+
+  render() {
+
+    const {} = this;
+
+    return html`
+    <main>
+    <h3>Product </h3>
+    <ul>
+    <column-list text_line="Manage" color="white" class="list" ></column-list>
```

Comment:
should be separated into STR_ and outside of the render function

Reply:
Fixed